pkg/security/secl/model/model.go

```diff
@@ -796,6 +797,16 @@ type DNSEvent struct {
 	Count uint16 `field:"question.count"`                              // the total count of questions in the DNS request
 }
 
+// BindEvent represents a bind event
+//msgp:ignore BindEvent
+type BindEvent struct {
+	SyscallEvent
+
+	Socket     int32         `field:"socket"`      // The socket to be bound
```

Comment:
Unless you thought of a use case, we should probably remove it for now: we don't export any file descriptor from any other event so I'm not sure what you can do with it.

Reply:
I'm not sure either. OK let's simplify it for now.

Reply:
Updated, thanks.